src/Elmish.WPF/BindingVmHelpers.fs

```diff
@@ -613,22 +614,24 @@ type Update
           this.Recursive(currentModel |> ValueOption.map b.Get, getCurrentModel >> b.Get, b.Get newModel, b.Dispatch, b.Binding)
 
 
-type Get(nameChain: string) =
+type Get<'t>(nameChain: string) =
 
-  member _.Base (model: 'model, binding: BaseVmBinding<'model, 'msg>) =
+  member _.Base (model: 'model, binding: BaseVmBinding<'model, 'msg, 't>) =
     match binding with
     | OneWay { OneWayData = d } -> d.Get model |> Ok
     | TwoWay b -> b.Get model |> Ok
     | OneWayToSource _ -> GetError.OneWayToSource |> Error
     | OneWaySeq { Values = vals } -> vals.GetCollection () |> Ok
-    | Cmd cmd -> cmd |> box |> Ok
-    | SubModel { Vm = vm } -> vm.Value |> ValueOption.toObj |> box |> Ok
+    | Cmd cmd -> cmd |> unbox |> Ok
```

Comment:
From https://github.com/elmish/Elmish.WPF/pull/470#discussion_r963186151:

> I was surprised by the use of unbox here. However, I tested and it works.
> 
> Is it possible avoid this call to unbox? My intuition is that it cannot be avoided, but I hope that I am wrong.

-- @TysonMN

Reply:
We *might* be able to move it further down with some redesign of the `VmBinding.Cmd` case. You would need a getter that just returns type `'t` and constrain the type on creation of the `BindingData.Cmd` case.